--- a/src/oc/hoc.c	Mon Apr 21 13:33:19 2014 -0400
+++ b/src/oc/hoc.c	Mon Apr 21 17:24:25 2014 -0400
@@ -122,6 +122,7 @@
 #endif
 
 #if READLINE
+extern char* readline(const char* prompt);
 extern void rl_deprep_terminal(void);
 extern void add_history(const char*);
 #endif
@@ -146,7 +147,7 @@
 #define CBUFSIZE 512
 HocStr* hoc_tmpbuf;
 HocStr* hoc_cbufstr;
-char* hoc_promptstr;
+const char* hoc_promptstr;
 static CHAR	*cbuf;
 CHAR	*ctp;
 int hoc_ictp;
@@ -1692,7 +1693,7 @@
 		}
 	}else{
 #if READLINE
-		if (nrn_fw_eq(fin, stdin) && nrn_istty_) { char *line, *readline(); int n;
+		if (nrn_fw_eq(fin, stdin) && nrn_istty_) { char *line; int n;
 #if INTERVIEWS
 #ifdef MINGW
 IFGUI